src/uct/cuda/cuda_copy/cuda_copy_md.c

```diff
@@ -388,14 +362,17 @@ static ucs_status_t
 uct_cuda_copy_mem_release_fabric(uct_cuda_copy_alloc_handle_t *alloc_handle)
 {
 #if HAVE_CUDA_FABRIC
+    CUmemGenericAllocationHandle *generic_handle =
+        (CUmemGenericAllocationHandle*)alloc_handle->generic_handle;
```

Comment:
because we declare generic_handle now as void*
```C
 56 typedef struct uct_cuda_copy_alloc_handle {
 57     CUdeviceptr                  ptr;
 58     size_t                       length;
 59     void                         *generic_handle;
 60 } uct_cuda_copy_alloc_handle_t;
```